corehq/apps/settings/views.py

```diff
@@ -384,6 +387,14 @@ def dispatch(self, request, *args, **kwargs):
         # this is only here to add the login_required decorator
         return super(TwoFactorPhoneSetupView, self).dispatch(request, *args, **kwargs)
 
+    def done(self, form_list, **kwargs):
+        """
+        Store the device and reload the page.
+        """
+        self.get_device(user=self.request.user, name='backup').save()
+        messages.add_message(self.request, messages.SUCCESS, "Phone number added!")
```

Comment:
this needs to be translated

Reply:
Done here:
https://github.com/dimagi/commcare-hq/pull/19574/commits/feabee0c28af933d292aa4e6ed05557a99642c63